api/tests/integration/infrastructure/repositories/campaign-to-join-repository_test.js

```
@@ -186,10 +215,95 @@ describe('Integration | Repository | CampaignToJoin', () => {
       await databaseBuilder.commit();
 
       // when
-      const canJoinCampaign = await campaignToJoinRepository.isCampaignJoinableByUser(campaignToJoin, userId);
+      try {
+        await campaignToJoinRepository.checkCampaignIsJoinableByUser(campaignToJoin, userId);
+      } catch (error) {
+        expect.fail(`"${error}" should not have been thrown`);
+      }
+    });
 
-      // then
-      expect(canJoinCampaign).to.be.true;
+    it('should return false when campaign is not multipleSendings but there is already a participation for user', async () => {
```

`it('should throw an error when campaign is not multipleSendings and there is already a participation for user', async () => {`